### PR TITLE
Bar stools

### PR DESCRIPTION
**Pull request by Konater**
Just a subtype of stools, sprite stolen from goon
stools

### CODE REVIEW

**Comment by PJB3005** [comment]
If the sprite is ACTUALLY stolen from goon it MUST go under the `/goon` folder due to licensing issues. Retain folder / file structure on goon too.

**Comment by Konater** [comment]
In that case I better close this then

**Comment by PJB3005** [comment]
What? Just move the sprite and you're fine.

**Comment by PJB3005** [comment]
https://github.com/d3athrow/vgstation13/tree/Bleeding-Edge/goon

**Comment by clusterfack** [comment]
PJB that is unnecessary are sprites are already under the same license as goon, you should know this

**Comment by PJB3005** [comment]
@clusterfack no they aren't, goon is CC-BY-NA-NC, we are CC-BY-NA.

**Comment by clusterfack** [comment]
Fuck off license-fag

**Comment by PJB3005** [comment]
Never

**Comment by clusterfack** [comment]
We literally branched from them in r4407 we have the same license

File: code/game/objects/items/stacks/stack_recipes.dm [hunk 1]
```diff
@@ -92,6 +92,7 @@ var/global/list/datum/stack_recipe/metal_recipes = list (
 		new/datum/stack_recipe/chair("lime comfy chair",   /obj/structure/bed/chair/comfy/lime,   2, one_per_turf = 1, on_floor = 1),
 		new/datum/stack_recipe/chair("teal comfy chair",   /obj/structure/bed/chair/comfy/teal,   2, one_per_turf = 1, on_floor = 1),
 		new/datum/stack_recipe/chair("stool",              /obj/item/weapon/stool												   ),
+		new/datum/stack_recipe/chair("bar stool",          /obj/item/weapon/stool/bar                                              ),
```
Comment (9600bauds) [inline code/game/objects/items/stacks/stack_recipes.dm:95]: Are you sure about this formatting?

**Comment by PJB3005** [comment]
Nope

https://github.com/goonstation/goonstation-2016/blob/master/LICENSE.md

**Comment by clusterfack** [comment]
At no point between r4407 and now has the license been changed in a way that could be considered legal

**Comment by PJB3005** [comment]
@clusterfack then sue goon if you want.

The bar stool sprites were definitely post r4407 so...

**Comment by clusterfack** [comment]
No @PJB3005 I mean on our side not theirs

File: code/game/objects/items/stacks/stack_recipes.dm (hunk 1 above)
Comment (Konater) [inline code/game/objects/items/stacks/stack_recipes.dm:95]: not really but I thought since it was pretty much a stool from above I could copy

**Comment by Zth--** [comment]
I like, merg pls

**Comment by Intigracy** [comment]
If we wanna argue semantics, goon is based off code _stolen_ from the original creator of ss13.

**Comment by ghost** [comment]
Let's port those anyways. Even in the event they get absolutely devastated because a SS13 codebase on the Internet grabbed their 32x32 pixels sprite of a round bar stool, the worst that will occur is limited drama until they cool down

It's not like we're stealing the three quarters of their codebase anyways, we don't need the vast majority of it, especially with the way it's coded

**Comment by gbasood** [comment]
@clusterfack Considering the amount of effort we need to take to keep goon things separate I don't see how it's a real issue that we keep this in the /goon folder.

**Comment by Shadowmech88** [comment]
Would Goon _really_ sue us just for not putting a bar stool sprite in the right directory?

**Comment by clusterfack** [comment]
It's not even a problem about whether they WOULD either they or us ALREADY have an illegal license that we're ALREADY violating.

**Comment by CptWad** [comment]
Nobody will do anything until there is actual money in this game (IE paid for/invested/etc.)
Why would you fight for the rights of a game that people play for free?
I see only two ways of shit actually turning into a legal battle

Exadv1 decides to make his own server and make it pay-to-play

We make our server pay-to-play

otherwise its not worth it to sue a bunch of poor faggots who code for a hobby

**Comment by gbasood** [comment]
Can we please keep it separate at least so we know it isn't ours?
I don't care about legal reasons, but it's nice to know what is from goon and what isn't.

**Comment by Konater** [comment]
So what should I do with this? When I tried to move the sprite to the goon folder when it was first suggested I got errors, I can try and make my own sprite if it will make everyone feel better

**Comment by ghost** [comment]
Just keep it as is

We've been integrating sprites from all sources for as long as we've had our own code, that's one more occurrence of it

**Comment by PJB3005** [comment]
@Konater what kind of errors?

**Comment by CptWad** [comment]
wait we have a port tag right

**Comment by Konater** [comment]
Uh I forget, it was something like rsc locking up

**Comment by PJB3005** [comment]
@Konater you probably had a server running in the background or something, DM dies if it has to recompile resources with a running server.

**Comment by Konater** [comment]
@PJB3005 oops

**Comment by ComicIronic** [comment]
Died to license issues?

**Pull request merged**